Activity.js

```diff
@@ -22,7 +22,15 @@ class Activity {
         var today = new Date();
         var distance = nye - today;
         if (distance < 0) {
-          alert('Activity complete. What\'s next?')
+          //KATIE PLEASE MOVE TO MAIN.JS FUNCTION FROM HERE TO....
```

Comment:
Hey team, great job getting the timer working, but I have some suggestions. 

First, on line 13, instead of using minutesInput.value & secondsInput.value, make sure to use the values actually stored in the class, `this.minutes` & `this.seconds`. Also, as an aside, `nye` is a confusing variable name; maybe something like `futureTime` might be more accurate.

Next, for better dev empathy, I suggest you move lines 15-18 down above line 38, as that's when those variable definitions are used. 

Lastly, it's important to make sure that your data model is separate from any DOM manipulation. This Activity class should only handle data manipulation, and should just return out any values that are then needed to change the display in main.js. For instance, lines 50 & 51 should not live inside the Activity class. I worry that the fact that you've housed all functionality related to your timer within the one startTimer function might make it difficult to separate out the DOM functionality.

Reply:
Awesome, thanks for your feedback Rachel! We are making a note to address these issues in the next push/merge.